package.json

```diff
@@ -38,10 +38,13 @@
     "@vue/cli-service": "4.3.1",
     "@vue/test-utils": "1.0.2",
     "autoprefixer": "9.7.6",
+    "babel-core": "7.0.0-bridge.0",
     "browser-sync": "2.26.7",
     "browser-sync-webpack-plugin": "2.2.2",
     "copy-webpack-plugin": "6.0.0",
     "css-loader": "3.5.3",
+    "cypress": "4.3.0",
```

Comment:
update to 4.8.0?

Reply:
We have an issue when we want to update to a version of Cypress > 4.3.
There is an error about tslib.
![cypress-error](https://user-images.githubusercontent.com/23552990/85204974-7d3da380-b318-11ea-9d15-7dba3705f611.png)

Reply:
It seems that is a problem with multiple tsconfig.json (one for jhipster control center, one for cypress) but we still no found a solution for the moment.